elastic_datashader/config.py

```diff
@@ -93,14 +94,15 @@ def config_from_env(env) -> Config:
         datashader_headers=load_datashader_headers(env.get("DATASHADER_HEADER_FILE", "headers.yaml")),
         elastic_hosts=env.get("DATASHADER_ELASTIC", "http://localhost:9200"),
         ellipse_render_mode=env.get("DATASHADER_ELLIPSE_RENDER_MODE", "matrix"),
+        ellipse_render_min_zoom=env.get("DATASHADER_ELLIPSE_RENDER_MIN_ZOOM", 8),
         hostname=getfqdn(),
         log_level=get_log_level(env.get("DATASHADER_LOG_LEVEL", None)),
         max_batch=int(env.get("DATASHADER_MAX_BATCH", 10_000)),
         max_bins=int(env.get("DATASHADER_MAX_BINS", 10_000)),
         max_ellipses_per_tile=int(env.get("DATASHADER_MAX_ELLIPSES_PER_TILE", 100_000)),
         max_legend_items_per_tile=int(env.get("MAX_LEGEND_ITEMS_PER_TILE", 20)),
         num_ellipse_points=int(env.get("DATASHADER_NUM_ELLIPSE_POINTS", 100)),
-        query_timeout_seconds=int(env.get("DATASHADER_QUERY_TIMEOUT", 0)),
+        query_timeout_seconds=int(env.get("DATASHADER_QUERY_TIMEOUT", 900)),
```

Comment:
To confirm, to change this timeout on the other network we would only need to change this config file and restart DS? 
This is the parameter that on the other network we had set to 3 minutes (and when set to 2 minutes that was determined to be too low)?

Reply:
Yes other network set this ENV variable to set the timeout param for the searches